interfaces/cython/cantera/yaml2ck.py

```diff
@@ -170,7 +170,7 @@ def build_species_text(species: Iterable[ct.Species], max_width=80) -> str:
 
     .. versionadded:: 3.0
     """
-    species_names = {s.name: s.input_data.get("note", "") for s in species}
+    species_names = {s.name: f"{s.input_data.get('note')}" for s in species}
```

Comment:
I prefer to be more explicit about casting to strings, with:
```suggestion
    species_names = {s.name: str(s.input_data.get('note', '')) for s in species}
```